network-api/networkapi/wagtailpages/tests/test_utils.py

```diff
@@ -17,15 +18,35 @@
 )
 from networkapi.wagtailpages.factory import blog as blog_factories
 from networkapi.wagtailpages.factory.profiles import ProfileFactory
-from networkapi.wagtailpages.pagemodels.blog.blog import BlogAuthors
+from networkapi.wagtailpages.pagemodels.blog.blog import BlogAuthors, BlogPage
 from networkapi.wagtailpages.pagemodels.profiles import Profile
 from networkapi.wagtailpages.utils import (
     create_wagtail_image,
     get_blog_authors,
+    get_content_related_by_tag,
     localize_queryset,
 )
 
 
+class TestGetContentRelatedByTag(TestCase):
```

Comment:
We need more tests here, but that would extend the scope of this ticket too much.